### PR TITLE
Implement raw field for instance variables API

### PR DESCRIPTION
**Pull request by timofurrer**

### CODE REVIEW

**Comment by timofurrer** [comment]
@svanharmelen do you mind giving this a review and release? 🙏

File: instance_variables.go [hunk 1]
```diff
@@ -107,6 +108,7 @@ type CreateInstanceVariableOptions struct {
 	VariableType *VariableTypeValue `url:"variable_type,omitempty" json:"variable_type,omitempty"`
 	Protected    *bool              `url:"protected,omitempty" json:"protected,omitempty"`
 	Masked       *bool              `url:"masked,omitempty" json:"masked,omitempty"`
+	Raw          bool               `url:"raw,omitempty" json:"raw,omitempty"`
```
Comment (svanharmelen) [inline instance_variables.go:111]: This needs to be a pointer...

File: instance_variables.go [hunk 2]
```diff
@@ -140,6 +142,7 @@ type UpdateInstanceVariableOptions struct {
...
```
Comment (svanharmelen) [inline instance_variables.go:145]: This needs to be a pointer...

**Review by svanharmelen — CHANGES_REQUESTED** [review]

**Review by svanharmelen — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**